shell.nix

```diff
@@ -1,4 +1,4 @@
-with import ./pin { snapshot = "nixos-20-03_0"; };
+with import ./pin { snapshot = "nixos-unstable_0"; };
```

Comment:
@sarcasticadmin so is "nixos-unstable_0" the name of the "current" nix repo or is it the name we defined when we "manually take a snapshot of the current nix repo" if its the former, thats freaking awesome. just thought it was too simple of a name to be able to distinguish between other snapshots 😅

Reply:
@jsnguyenwork good question! `nixos-unstable_0` refers to a point in time snapshot of the nixpkg-channel's unstable branch: https://github.com/nixos/nixpkgs-channels there potentially could be many of these used by different parts of this repo. This idea is that this guarantees whatever is sourcing a snapshot will work the same every time until its deliberately bumped to another pin. I want to keep the naming simple since these pins are just pointers to commits on the nix-channel, thats it.